indico/modules/events/papers/views.py

```diff
@@ -16,22 +16,28 @@
 
 from __future__ import unicode_literals
 
+from indico.modules.events.abstracts.views import _MathjaxMixin
```

Comment:
Would you also move the `util.py` from `events/util.py` inside the `events/util/` folder? And maybe rename it to `util/base.py`?